index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <h1>Welcome</h1>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
+      </header>
+      <section class="top-header">
+        <div class="header-title">
+          <h2>Hey there. I'm Bamsi</h2>
+          <h2 class="color-green">I'm a software developer</h2>
+        </div>
+        <div class="header-body">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
```

Comment:
- [ ] Kindly adjust the color of this section to match the Figma template
![review_image1](https://user-images.githubusercontent.com/69858269/158909878-68f66f05-fd97-4e74-b1a6-c2c9d5c8f0b0.png)